models/stg_hubspot__ticket.sql

```diff
@@ -29,7 +29,7 @@ with base as (
         }}
         {% if all_passthrough_column_check('stg_hubspot__ticket_tmp',get_ticket_columns()) > 0 %}
         -- just pass everything through if extra columns are present, but ensure required columns are present.
-        ,{{  
+        {{  
```

Comment:
Out of curiosity, is the reason this is not needed in the other models like `deal`, `contact`, and `company` due to the fact that we do not use the remove_duplicate macro? I mainly ask because I see the same comma there, but the absence of the new macro is the only reason I could think for why we are not including an update for those.

Reply:
Yes, it is because of [line 9 (and similar)](https://github.com/fivetran/dbt_hubspot_source/blob/028e1421c71caf588296c81bfc6b0b81cc0690b3/macros/remove_duplicate_and_prefix_from_columns.sql#L9) of `remove_duplicate_and_prefix_from_columns`. 

The outputs end up being different since this macro has a comma proceeding the entries, while `fivetran_utils.remove_prefix_from_columns` [line 9](https://github.com/fivetran/dbt_fivetran_utils/blob/releases/v0.4.latest/macros/remove_prefix_from_columns.sql#L9) puts it at the end if not the end of the loop.

Reply:
Thanks for the detailed explanation! This sounds good to me 😄